packages/services/api/src/modules/schema/resolvers.ts

```diff
@@ -458,16 +435,47 @@ export const resolvers: SchemaModule.Resolvers = {
     },
   },
   SchemaVersion: {
-    commit(version, _, { injector }) {
-      return injector.get(SchemaManager).getCommit({
+    async log(version, _, { injector }) {
+      const log = await injector.get(SchemaManager).getSchemaLog({
         commit: version.commit,
         organization: version.organization,
         project: version.project,
         target: version.target,
       });
+
+      if (log.kind === 'single') {
+        return {
+          __typename: 'PushedSchemaLog',
+          author: log.author,
+          commit: log.commit,
+          date: log.date as any,
```

Comment:
You can select the date as a string from the database. with `to_json("fieldname") as "date"`. This is actually the better way of loading the date because if it is loaded as a number it will be inaccurate. E.g. if you load the date as number and then try to query for the record with that number you would get zero results.  See https://tortilla-hq.slack.com/archives/CAY2119MX/p1674036685685139

Reply:
Add it as a task to change it everywhere